tests/beaker_tests/lib/utilitylib.rb

```diff
@@ -1029,3 +1029,39 @@ def find_interface(tests, id=nil, skipcheck=true)
   end
   intf
 end
+
+# Use puppet resource to get interface capability information.
+# TBD: Facter may be a better home for this method but the performance hit
+# appears to be 2s per hundred interfaces so it works better for now as an
+# on-demand method.
+def interface_capabilities(agent, intf)
+  cmd = get_namespace_cmd(agent, PUPPET_BINPATH +
```

Comment:
Not sure how you want to handle this, but after https://github.com/cisco/cisco-network-puppet-module/pull/233 gets merged I believe you no longer need to call `get_namespace_cmd`.